src/main/java/core/basesyntax/PasswordValidationException.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class PasswordValidationException extends RuntimeException {
```

Comment:
should you extend Exception or RuntimeException? is this exception checked or unchecked?

Reply:
ok, I understood. I have to use Exception because my own exception is checked.